test/2_management_api/correlations/get_all_correlations.js

```diff
@@ -76,6 +78,34 @@ describe('Management API:   GET  ->  /correlations/all', () => {
     });
   });
 
+  it('should include correlations that were finished with an error', async () => {
+    try {
+      await createFinishedProcessInstance(testFixtureProvider.identities.defaultUser, errorProcessModelId);
+
+      should.fail('The expected Error was not thrown');
+    } catch (error) {
+      /**
+      * Give the persistance backend some time to persist the Correlation
+      * results.
+      */
+      await wait(750);
```

Comment:
Pretty sure half a second suffices here.

Please run this test a couple of times with 

```suggestion
      await wait(500);
```

if it turns out, that half a second doesn't suffice after all, you can set this to 

```suggestion
      await wait(1000);
```

But I'd recommend to always use 500ms steps, to make the number look less arbitrary.